File: runtimes/php-7.1/actions/helloworld.php
<?php
/**
  *
  * main() will be invoked when you Run This Action.
  *
  * @param Cloud Functions actions accept a single parameter,
  *        which must be a JSON object.
  *
  * In this case, the params variable will look like:
  *     { "name": "xxxx" }
  *
  * @return which must be a JSON object.
  *         It will be the output of this action.
  *
  */
function main(array $args) : array
{
  $name = $args["name"] ?? "stranger";
  $greeting = "Hello $name!";
  echo $greeting;
  return ["greeting" => $greeting];
}

File: runtimes/python-3.6.4/actions/helloworld.py
#
#
# main() will be invoked when you Run This Action.
#
# @param Cloud Functions actions accept a single parameter,
#        which must be a JSON object.
#
# @return which must be a JSON object.
#         It will be the output of this action.
#
#
import sys

def main(dict):
  if 'name' in dict:
    name = dict['name']
  else:
    name = 'stranger'
  greeting = 'Hello ' + name + '!'
  print(greeting)
  return {'greeting':greeting}
